app/Config/Routing.php

```diff
@@ -26,7 +26,7 @@ class Routing extends BaseRouting
      * Default: APPPATH . 'Config/Routes.php'
      */
     public array $routeFiles = [
-        APPPATH . 'Config/Routes.php',
+        APPPATH . 'Config' . DIRECTORY_SEPARATOR . 'Routes.php',
```

Comment:
If we use this way, we would need to use `DIRECTORY_SEPARATOR` in all file path strings.
To be honest, I don't want to write such code.

Reply:
@kenjis I think the best way is handling normalize the `routeFile` array property in constructor.